doc/DESI_TARGET/SCND_DIR/PHASE/outdata/VERSION/OBSCON/BITNAME.rst

```diff
@@ -0,0 +1,99 @@
+========================
+output secondary targets
+========================
+
+:Summary: DESI secondary target output files include a binary fits table. This table
+	  contains the targets provided with a given secondary proposal, with the
+	  added targeting information (`i.e.` IDs and bits) needed by the DESI pipeline.
+:Naming Convention: ``BITNAME.fits``,
+   where ``BITNAME`` matches the name of the secondary targeting bit used by the desitarget
+   pipeline for a given survey phase (see the desitarget GitHub repository for, `e.g.`
+   the `sv1`_ or `main`_ secondary target bitmasks).
+:Regex: ``[a-zA-Z]+\.fits``
```

Comment:
Aren't bit names capitalized? In which case ``[A-Z]+`` would be sufficient?  And they never, ever contain numbers?

Reply:
Looking at the actual bitmask files in `ets`, it is clear that `[A-Z_0-9]+` is needed to match all of the files.

However, there is another file that appears in the `outdata` directories: `veto.fits`.  Is that a separate file type?